### DISCUSSION TITLE
email loads extremely slow in 6.1.0

### DISCUSSION THREAD

**Discussion by simple-AMS** [YetiForceCompany/YetiForceCRM#13965] [General]
Hi @mariuszkrzaczkowski,

You asked me to place #13962 here instead so my apologies for placing my question in the wrong area. I did update the system as I stated in my original question. I have installed Service Packs 1, 2 and 3 and the email is still loading terribly slow. I have absolutely no errors anywhere and this is the first version that I have ever had this issue with. Any suggestions?

**Comment by simple-AMS** [comment]
@mariuszkrzaczkowski Anything??

**Reply by mariuszkrzaczkowski** [reply]
comment this line
https://github.com/YetiForceCompany/lib_roundcube/blob/9a2d2e8525d41da615e6e67fe4b9885662950b85/plugins/yetiforce/yetiforce.php#L32

**Comment by simple-AMS** [comment]
Hi @mariuszkrzaczkowski, Gave that a try but sadly the issue persists. Any other possibilities?

**Comment by simple-AMS** [comment]
Hi @mariuszkrzaczkowski,

I can send emails without a problem. The issue is it will never load the Inbox messages. I have set up the Roundcube configuration and SMTP configuration with the right server settings for Office365 email service but it simply WILL NOT connect with my inbox. 

Check out the screenshots below:

This one is where I select the INBOX where it's clearly showing that I have 6 new messages. As you can see in the bottom right hand corner, it's loading and it seems to do that forever.

![Screenshot from 2020-12-31 23-04-20](https://user-images.githubusercontent.com/42642523/103433803-51e81b00-4bbd-11eb-9571-83891da61d0a.png)

This one shows a server error, every single time. Yet I can click on all the other folders and you can see in the last screenshot.

![Screenshot from 2020-12-31 23-06-18](https://user-images.githubusercontent.com/42642523/103433807-59a7bf80-4bbd-11eb-90be-c778dee6defb.png)

See how it shows there's no messages found? Yet I can click on the Sent folder and it shows the messages that I have sent out from there. Yet on the inbox it just loads and loads forever and always fails with a server error?

![Screenshot from 2020-12-31 23-07-07](https://user-images.githubusercontent.com/42642523/103433810-5dd3dd00-4bbd-11eb-8382-1e374d7717a8.png)

Please help @mariuszkrzaczkowski and thank you!

**Comment by simple-AMS** [comment]
@mariuszkrzaczkowski Help please :)

**Comment by KatarzynaUlichnowska** [comment]
It needs to be verified by a developer as it's a performance issue. One of our developers can do it for you but you need to buy one of the packages https://yetiforce.com/en/marketplace/support

**Reply by simple-AMS** [reply]
Hi @KatarzynaUlichnowska,

Thanks for your reply. I'm not a software developer but I am a server administrator. I can assure you, it's not a performance issue. It's not even a software issue on your end. It's something on Roundcube's end I believe. Since however the Yeti development teamed forked it for use within the system, I assumed they would know where to look :)

I can't be the only one having this issue though so just wanted to bring it up because clients that have Microsoft Office365 Email accounts will experience this issue and there are a lot of people that use that service. They will all have this issue, I promise you.

**Comment by scotlad9** [comment]
I'm evaluating Yetiforce 6.1 (service pack 7) and have similar issues with fetching emails from an Office 365 account.  On the same server, an installation of Roundcube works fine but in Yetiforce it times out retrieving the inbox unless I change the sort order to ascending.  Then it displays the oldest 30 emails (out of about 200) but if I try to page to the previous 30 it times out again.  Since Roundcube is working as a separate application, I conclude there must be something different in the Yetiforce implementation - help please as this is a show-stopper for us.

**Reply by mariuszkrzaczkowski** [reply]
See my post here https://github.com/YetiForceCompany/YetiForceCRM/issues/14489#issuecomment-812328931

**Reply by simple-AMS** [reply]
@scotlad9 Dude, thank God for you man. Had you not said something, I would have NEVER got a solution for this problem. I have been waiting for the longest time for this simple solution that now makes the system I was about to ditch finally work as it should. Thanks a million bro!

**Reply by scotlad9** [reply]
Hope you're able to stick with it till 6.2 arrives.  I've been using VTiger for years but now thinking of switching to YetiForce because the VTiger Outlook plug in I use to link emails has stopped working properly and it's been deprecated.

**Reply by simple-AMS** [reply]
@scotlad9 I always do. With that said, every time a new version comes out, I end up with issues like this that finally get resolved right before a new version is released and then I have to go through the same thing all over again. Talk about the definition of insanity.
